docker-compose.yml

```diff
@@ -1,20 +1,26 @@
-db:
-  image: mysql:5.6
-  environment:
-    - MYSQL_DATABASE=mapknitter
-    - MYSQL_USER=mapknitter
-    - MYSQL_PASSWORD=mapknitter
-    - MYSQL_RANDOM_ROOT_PASSWORD=true
-  volumes:
-    - /srv/mapknitter_priv1/mk1_database:/var/lib/mysql
```

Comment:
This should be a relative path (e.g. `../mysql`)

Reply:
Docker does not accept relative path there's no way of your host knowing there is or is not in the same server as your client. See https://github.com/moby/moby/issues/4830#issuecomment-38488498